bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml

```diff
@@ -34,6 +34,9 @@
 			<channel id="fan" typeId="fan"/>
 			<channel id="vacuum" typeId="vacuum"/>
 			<channel id="segment" typeId="segment"/>
+			<channel id="collectdust" typeId="collectdust"/>
+			<channel id="cleanmopstart" typeId="cleanmopstart"/>
+			<channel id="cleanmopstop" typeId="cleanmopstop"/>
```

Comment:
See below:
```suggestion
			<channel id="collect_dust" typeId="collect_dust"/>
			<channel id="clean_mop_start" typeId="clean_mop_start"/>
			<channel id="clean_mop_stop" typeId="clean_mop_stop"/>
```

Reply:
Yeah, thanks same here: I would better had have a look on how you writing stuff in here